### DISCUSSION TITLE
Submission Error [Pending Timeout]

### DISCUSSION THREAD

**Discussion by wizzniu** [PaddlePaddle/PaddleSpatial#260] [BaiduKDDCup2022]
Team name: wizzTeam. My submission at 2022/7/13 23:14 encountered some error. It says "Pending Timeout". I wanna know what's wrong? 
![image](https://user-images.githubusercontent.com/39826673/178857475-bb199d4e-2072-4f4d-9d22-02ff377d6576.png)

**Comment by seeingisbelieveing** [comment]
The submission 2022-07-13 23:14:11 displayed  "异常"， please resubmit it and run it again.

**Comment by ramber1836** [comment]
This error means that the pending time of your submission is too long and thus your submission is killed by the system.

**Reply by wizzniu** [reply]
Thanks, I have resubmitted it again.
